mlrun/model_monitoring/writer.py

```diff
@@ -201,4 +223,12 @@ def do(self, event: _RawEvent) -> None:
         self._update_tsdb(event)
         self._update_kv_db(event)
         _Notifier(event=event, notification_pusher=self._custom_notifier).notify()
```

Comment:
yes, after a conversation with @jond01 , it was agreed that it will be deleted after this PR is merged

Reply:
No problem, please erase this line from the writer, and about the `_Notifier` class @jond01 will erase it with all the relevant tests in separate pr.

Reply:
> yes, after a conversation with @jond01 , it was agreed that it will be deleted after this PR is merged

@Yacouby - that is not exactly what we talked about IIRC.
Since the alerts will be disabled by default in the coming 1.7, we will remove the custom notifier in the writer only in 1.8.